Posts/2021/03/File-System-Watcher-Engine-Event.md

```diff
@@ -66,45 +78,39 @@ PS> "XYZ" >> C:\temp\files\xyz
 }
 ```
 
-Also event raised before a handler is registered will remain in the queue until they are consumed
-manually using '[Get-Event](xref:Microsoft.PowerShell.Utility.Get-Event)'/'[Remove-Event](xref:Microsoft.PowerShell.Utility.Remove-Event)'. This has nothing to do with this module and is just the
-way PowerShell works.
+Events raised before a handler is registered remain in the queue until you consume them using '[Get-Event](xref:Microsoft.PowerShell.Utility.Get-Event)'/'[Remove-Event](xref:Microsoft.PowerShell.Utility.Remove-Event)'.
 
 To suspend the notification temporarily and to resume it later the following two commands can be used:
 
 ```powershell
-PS> Suspend-FileSystemWatcher -SourceIdentifier "myevent"
+Suspend-FileSystemWatcher -SourceIdentifier "MyEvent"
 
-PS> Resume-FileSystemWatcher -SourceIdentifier "myevent"
+Resume-FileSystemWatcher -SourceIdentifier "MyEvent"
 ```
 
-To keep track of all the filesystem watchers created in the current Powershell process the command
-'Get-FileSystemWatcher' can be used:
+To keep track of all the filesystem watchers created in the current Powershell process, you can use the command 'Get-FileSystemWatcher':
 
-```powershell
+```powershell-console
 PS>  Get-FileSystemWatcher
 
-SourceIdentifier      : myevent
-Path                  : C:\temp\files\
+SourceIdentifier      : MyEvent
+Path                  : C:\Temp\files\
 NotifyFilter          : FileName, DirectoryName, LastWrite
 EnableRaisingEvents   : True
 IncludeSubdirectories : False
 Filter                : *
 ```
 
-It will write a state object to the pipe containing the configuration of all filesystem watchers.
-Finally if you want to ger rid of all filesystem watchers the command 'Remove-FileSystemWatcher' will
-dispose a filesystem watcher specified by the source identifier of by piping in a collection of
-watchers:
+This command write a state object to the pipe containing the configuration of all filesystem watchers.
```

Comment:
This command writes a state object to the pipe containing the configuration of all filesystem watchers.

Reply:
fixed